integration/src/test/resources/org/mule/test/integration/notifications/message-processor-notification-test-flow.xml

```diff
@@ -262,8 +262,8 @@
 
     <flow name="composite-source">
         <composite-source>
-            <custom-source class="org.mule.test.core.context.notification.processors.MessageProcessorNotificationTestCase$TestMessageSource"/>
-            <custom-source class="org.mule.test.core.context.notification.processors.MessageProcessorNotificationTestCase$TestMessageSource"/>
+            <test:custom-source class="org.mule.test.core.context.notification.processors.MessageProcessorNotificationTestCase$TestMessageSource"/>
```

Comment:
can you put the logic from this source into `SkeletonSource` to avoid having this class?